src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,37 @@
+package duke;
+
+import duke.command.*;
```

Comment:
You might want to import the classes individually instead of importing all at once